tests/integration/src/test/resources/org/mule/test/integration/notifications/modules/module-simple.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module name="module-simple"
+        minMuleVersion="4.1.0"
+        prefix="simple-prefix"
+        namespace="http://www.mulesoft.org/schema/a/different/path/mule/module-simple"
+        xmlns="http://www.mulesoft.org/schema/mule/module"
+        xmlns:mule="http://www.mulesoft.org/schema/mule/core"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="
+           http://www.mulesoft.org/schema/mule/module http://www.mulesoft.org/schema/mule/module/current/mule-module.xsd
+           http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
+    <operation name="set-payload-hardcoded-value">
+        <body>
+            <mule:set-payload value="hardcoded value"/>
```

Comment:
what would happen if there are two modules `a` and `b`, both of them exposing the same `set-payload-hardcoded-value` operation and I have an app that consumes both of them in some flows (`<a:set-payload-hardcoded-value../>` and `<b:set-payload-hardcoded-value..>`), wouldn't using the name of the operation as root of the stack flow be inconvenient?